README.md

```diff
@@ -1,14 +1,11 @@
-# Perfect
-The best Next.js boilerplate ever.
-# What does this have?
-- Tailwind CSS (no more going through the installation guide!) - for styling components easily
-- `next-themes` (@pacocoursey) - for going to the dark side
-- `swr` (@vercel) - for fetching constantly mutating data
-  - `fetcher.js` so you don't have to keep writing return functions
-- `framer-motion` - for making interactive components and eye-catching animations
-- `react-icons` - for including your favorite icon packs without increasing bundle size significantly. Tree shaking included.
-- `classnames` - for conditionally applying classes to components
-- `react-wrap-balancer` - for balancing text widths for titles and paragraphs
-- `tailwind-merge` - for merging Tailwind class names
-
-&copy; 2021 Sarthak Mohanty. Licensed under the [Unlicense](LICENSE) from Mozilla.
+# Ham Club - A New YSWS from Hack Club
+Ham Club is a program for teens in the US running from April 29th to May 31st, allowing youth to get their ham radio license and then receive a free ham radio!
+
+# You Ship, We Ship
+
+At Hack Club we're re-imagining a new type of public education for technology. We believe people learn best by creating things they care about and sharing them with others. We build open source creative tools with young people all around the world. These tools are gateways to new subjects in technology like [embedded systems](https://sprig.hackclub.com), [circuits](https://onboard.hackclub.com) or digital fabrication robotics. When teenagers build things with our tools we send them more creative materials for free. 
```

Comment:
@crnicholson I commented on this before... is this the official YSWS wording for marketing? It feels very AI written right now.

Reply:
Yep this is from Blot.

Reply:
Where? I don't see it?

Reply:
<img width="887" alt="image" src="https://github.com/hackclub/ham-club/assets/83499056/ead5b8a3-1d2f-4251-bd77-f2ed4aa5e2de">

https://github.com/hackclub/blot